server/app/api/controllers/self_review.js

```diff
@@ -0,0 +1,125 @@
+const Self_Review_Model = require("../models/self_review");
+module.exports = {
+  create: function(req, res, next) {
+    Self_Review_Model.create(
+      {
+        employee_id: req.body.employee_id,        
```

Comment:
spread operator

Reply:
Destructuring ? you mean . This is done.